src/backend/distributed/sql/udfs/citus_worker_stat_activity/latest.sql

```diff
@@ -1,19 +1,9 @@
-DROP FUNCTION IF EXISTS pg_catalog.citus_worker_stat_activity CASCADE;
+DROP VIEW IF EXISTS pg_catalog.citus_worker_stat_activity;
 
-CREATE OR REPLACE FUNCTION pg_catalog.citus_worker_stat_activity(OUT query_hostname text, OUT query_hostport int, OUT distributed_query_host_name text, OUT distributed_query_host_port int,
-                                                      OUT transaction_number int8, OUT transaction_stamp timestamptz, OUT datid oid, OUT datname name,
-                                                      OUT pid int, OUT usesysid oid, OUT usename name, OUT application_name text, OUT client_addr INET,
-                                                      OUT client_hostname TEXT, OUT client_port int, OUT backend_start timestamptz, OUT xact_start timestamptz,
-                                                      OUT query_start timestamptz, OUT state_change timestamptz, OUT wait_event_type text, OUT wait_event text,
-                                                      OUT state text, OUT backend_xid xid, OUT backend_xmin xid, OUT query text, OUT backend_type text, OUT global_pid int8)
-RETURNS SETOF RECORD
-LANGUAGE C STRICT AS 'MODULE_PATHNAME',
-$$citus_worker_stat_activity$$;
+CREATE OR REPLACE VIEW citus.citus_worker_stat_activity AS
+SELECT * FROM citus_stat_activity
+WHERE is_worker_query = true
+AND  backend_type = 'client backend';
```

Comment:
Can we completely drop citus_worker_stat_activity and only create it for the tests?

Because citus_worker_stat_activity doesn't give much interesting information, citus_stat_activity is often more than enough for such detailed investigation.

Also, we cannot use `backend_type` because it is NULL for privileged users, which makes it even harder to allow users to rely on this view. If we really want to show only the activity on the shards, we should either (a) make is_worker_query work properly (e.g.. NULL for PG related backends) (b) maybe use gpid?

Anyway, lets not get into such complexity?

Reply:
Updated tests with `citus_worker_stat_activity` to use `citus_stat_activity` instead.